.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    tags:
+
+jobs:
+  matrix:
```

Comment:
Does the matrix check `diff` to see if unit tests need to be executed if no PHP file has changed?
Documentation updates should not trigger anything imho.

Reply:
It doesn't currently, but it's something we can address in the laminas-ci-matrix-action (I was just mentioning this to @Ocramius a short bit ago). If we go that route, the action will basically provide an empty matrix, so no steps would be performed. (Even better: we might actually add functionality for linting the markdown, so such pushes would create a matrix that provides a job for that.)